packages/admin/admin/src/common/FieldSet.tsx

```diff
@@ -0,0 +1,147 @@
+import ArrowForwardIosSharpIcon from "@mui/icons-material/ArrowForwardIosSharp";
+import { Theme } from "@mui/material";
+import MuiAccordion, { AccordionProps as Props } from "@mui/material/Accordion";
+import MuiAccordionDetails, { AccordionDetailsProps } from "@mui/material/AccordionDetails";
+import MuiAccordionSummary, { AccordionSummaryProps } from "@mui/material/AccordionSummary";
+import { createStyles, WithStyles, withStyles } from "@mui/styles";
+import clsx from "clsx";
+import * as React from "react";
+
+interface FieldSetComponentsProps {
+    fieldSetSummary?: Partial<AccordionSummaryProps>;
+    fieldSetDetails?: Partial<AccordionDetailsProps>;
```

Comment:
@jamesricky do we really need this components props? And shouldn't the accordion be included as well?

Reply:
Generally I think we should allow overriding of the props of all components we use.  
Some projects require very specific customizations that are not possible (or very complicated) when the props cannot be overridden.

Reply:
@jamesricky would you add the Accordion's props as well? If yes, how would you call it? `root` or `fieldSet`? Also, should the other props be prefixed with fieldSet? Seems unnecessary to me...

Reply:
@jamesricky Can you comment on that?

Reply:
I think the props of every MUI-Component used should be overridable. 
I'd name them based on either the component's name or the `className` given. 
E.g.: `accordion`, `summary` & `details`